### DISCUSSION TITLE
5.2.0 release schedule

### DISCUSSION THREAD

**Discussion by Parzival2c** [xetorthio/jedis#3814] [General]
May I know if there's schedule of 5.2.0 formal release ? A rought time range will help as well. Thanks a lot.

**Comment by uglide** [comment]
Hey @Parzival2c 
We don't have any specific ETA yet since we are collecting feedback about the [Client-Side Caching.](https://github.com/redis/jedis/releases/tag/v5.2.0-beta1)

**Reply by Parzival2c** [reply]
Thanks a lot ,we've adopt 5.2.0 beta1 release in our testing envionrment for a while and test with positive results.
